concourse/tasks/tag-repo.yml

```diff
@@ -28,6 +28,7 @@ run:
 
       echo 'Configuring .ssh/config'
       echo "${GIT_SSH_PRIVATE_KEY}" > private-key
+      chmod 400 private-key
```

Comment:
Might this be better as 600 when considering any rotations?

Reply:
This is inside a pipeline, the volume gets deleted immediately after